### PR TITLE
feat: compose file with multiple nodes

### PR DESCRIPTION
**Pull request by smuu**
Closes https://github.com/celestiaorg/devops/issues/214

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -48,6 +49,21 @@ To use the rollup on the testnet, follow these steps:
...
+
+To use the rollup, follow these steps:
+
+1. Navigate to the directory containing the files.
+2. Run `docker compose -f multi.docker-compose.yaml build --no-cache` to build the rollup nodes.
+3. Run `docker compose -f multi.docker-compose.yaml up` to start the rollup and Celestia nodes.
+4. The rollup should now be accessible at `localhost:26657`.
+
+> If you have issues with the rollup, you can run `docker compose -f multi.docker-compose.yaml down` to stop the rollup and Celestia nodes and then run `docker compose -f multi.docker-compose.yaml build --no-cache` and `docker compose -f multi.docker-compose.yaml up` again to restart the rollup.
+> It's important to build the rollup again after stopping it, as this will generate a random namespace ID for the rollup. If you don't build the rollup again, the rollup will use the same namespace ID as before, which will cause issues.
```
Comment (Bidon15) [inline docker/README.md:62]: Thanks for the notes here 👍 
Do these issues get documented in rollkit?

**Review by Bidon15 — COMMENTED** [review]
Thank you for the work!

File: docker/multi.docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
+version: "3"
+
+services:
+  celestia:
+    image: ghcr.io/rollkit/local-celestia-devnet:v0.8.2
```
[the remark this answers is not in the document]
Comment (smuu) [reply docker/multi.docker-compose.yaml:5]: I will bump it to `v0.9.1` as it continues to work despite the errors I encountered.

File: docker/entrypoint.sh [hunk 3]
```diff
@@ -3,21 +3,26 @@
...
+if [ -z "$DA_BLOCK_HEIGHT" ]; then
+    DA_BLOCK_HEIGHT=$(curl --silent ${RPC%/}${RPC:+/}block | jq -r '.result.block.header.height')
+fi
 
+# If DA_BLOCK_HEIGHT not set manually or fetched from RPC, exit
 if [ -z "$DA_BLOCK_HEIGHT" ]; then
     echo "DA_BLOCK_HEIGHT is empty; ensure that the DA Layer is running and accessible"
     exit 1
 fi
+echo "DA_BLOCK_HEIGHT: $DA_BLOCK_HEIGHT"
```
Comment (jrmanes) [inline docker/entrypoint.sh:26]: I would add curly brackets to the variable as a best practices:

```suggestion
echo "DA_BLOCK_HEIGHT: ${DA_BLOCK_HEIGHT}"
```

Comment (smuu) [reply docker/entrypoint.sh]: done

**Review by Bidon15 — APPROVED** [review]
LGTM 🚀

**Review by jrmanes — APPROVED** [review]
LGTM 🚀

**Pull request merged**